blocks/article-body-block/chains/article-body/default.jsx

```diff
@@ -63,7 +64,13 @@ function parseArticleItem(item, index, arcSite, phrases, id, customFields) {
 		}
 
 		case "divider": {
-			return <hr className={`${BLOCK_CLASS_NAME}__divider`} key={`${type}_${index}_${key}`} />;
+			return (
+				<Divider
+					assistiveHidden={false}
+					className={`${BLOCK_CLASS_NAME}__divider`}
```

Comment:
We shouldn't need to add a `className` here as this is a component within the article body which is style-able via the components styles section of a block

Reply:
Fixed